<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_Multibanco</class>
 <widget class="QWidget" name="form_Multibanco">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>660</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>660</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Multibanco</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>imagens/multibanco1.png</normaloff>imagens/multibanco1.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #B6BBC4;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>491</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;, style=&quot;background-color:black&quot;,
font-family: &quot;quicksand, courier new, lucida console&quot;&gt;
MULTIBANCO&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>410</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>                                             SAÍDA DE DINHEIRO</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>                   SAÍDA DE TALÕES</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>        INTRODUZA O CARTÃO</string>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_ponto">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>590</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>550</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>510</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>470</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_seta1">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>180</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>imagens/green_arrowR .png</normaloff>imagens/green_arrowR .png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_pagamentos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>imagens/green_arrow.png</normaloff>imagens/green_arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>24</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>121</width>
     <height>8</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>632</x>
     <y>130</y>
     <width>121</width>
     <height>8</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>392</x>
     <y>430</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>550</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_0">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>590</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>470</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>510</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>550</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_00">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>590</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_9">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>470</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_6">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>510</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_anular">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>470</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
	color: red;
}</string>
   </property>
   <property name="text">
    <string>ANULAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_corrigir">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>510</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
	color: yellow;
}</string>
   </property>
   <property name="text">
    <string>CORRIGIR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_confirmar">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>550</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
	color: green;
}</string>
   </property>
   <property name="text">
    <string>CONFIRMAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_depositos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>imagens/green_arrow.png</normaloff>imagens/green_arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>24</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_consultas">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>imagens/green_arrow.png</normaloff>imagens/green_arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>24</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_levantamentos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>imagens/green_arrow.png</normaloff>imagens/green_arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>24</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_consultarIBAN">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>240</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>imagens/green_arrowR .png</normaloff>imagens/green_arrowR .png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_seta3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>300</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>imagens/green_arrowR .png</normaloff>imagens/green_arrowR .png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_codigo">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>360</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>imagens/green_arrowR .png</normaloff>imagens/green_arrowR .png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lblMultibanco">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>61</y>
     <width>491</width>
     <height>341</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>491</width>
     <height>341</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>22</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color:  #3171CE;
	font: 22pt &quot;Arial&quot;;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultibanco_voltar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>590</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
	color: black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lblMultibanco</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>btnMultibanco_ponto</zorder>
  <zorder>btnMultibanco_1</zorder>
  <zorder>btnMultibanco_4</zorder>
  <zorder>btnMultibanco_7</zorder>
  <zorder>btnMultibanco_seta1</zorder>
  <zorder>btnMultibanco_pagamentos</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>btnMultibanco_2</zorder>
  <zorder>btnMultibanco_0</zorder>
  <zorder>btnMultibanco_8</zorder>
  <zorder>btnMultibanco_5</zorder>
  <zorder>btnMultibanco_3</zorder>
  <zorder>btnMultibanco_00</zorder>
  <zorder>btnMultibanco_9</zorder>
  <zorder>btnMultibanco_6</zorder>
  <zorder>btnMultibanco_anular</zorder>
  <zorder>btnMultibanco_corrigir</zorder>
  <zorder>btnMultibanco_confirmar</zorder>
  <zorder>btnMultibanco_depositos</zorder>
  <zorder>btnMultibanco_consultas</zorder>
  <zorder>btnMultibanco_levantamentos</zorder>
  <zorder>btnMultibanco_consultarIBAN</zorder>
  <zorder>btnMultibanco_seta3</zorder>
  <zorder>btnMultibanco_codigo</zorder>
  <zorder>btnMultibanco_voltar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
